sw/device/tests/BUILD

```diff
@@ -5105,7 +5105,8 @@ opentitan_test(
         test_harness = "//sw/host/tests/chip/uart:uart_tx_rx",
     ),
     exec_env = {
```

Comment:
You need to create a `broken = new_cw310_params(tags = ["broken"])` entry

Reply:
It already exists for the _rom_ext exec env

Reply:
Ah sorry I missed it!